tractor/_spawn.py

```diff
@@ -167,8 +166,11 @@ async def spawn_subactor(
         # Hardcode this (instead of using ``_child.__name__`` to avoid a
         # double import warning: https://stackoverflow.com/a/45070583
         "tractor._child",
+        # This is merely an identifier for debugging purposes when
+        # viewing the process tree from the OS
```

Comment:
Not sure if this is accurate, I mean for sure seeing the uid in the proc tree is useful for debugging but, also its needed for the `tractor._child` initialization.

Reply:
Ahh, I think my point was that otherwise we'd just pass it over the channel from the parent.
Maybe I should clarify that *we only pass this on cmdline because..* ?

Reply:
That sounds like it would be clearer.

Reply:
Fixed :smile_cat: